extension.meta.xml

```diff
@@ -14,6 +14,9 @@
 		</author>
 	</authors>
 	<releases>
+		<release version="1.7.2" date="2017-05-19" min="2.5.x" max="4.x.x" php-min="5.6.x" php-max="7.x.x">
```

Comment:
I am glad that you remember this — I was afraid that I only dreamed. :-)

Hmmm, I already adopted it, e.g. here: https://github.com/michael-e/email_newsletter_manager/commit/c396584e9b3b2d8ecabecdb4cf3bdcaca1eec699

What should I do? Remove it? Or simply keep it?